dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -1396,17 +1371,13 @@ private boolean isAccessAuthorized(DataFile df, String apiToken) {
             } catch (WrappedResponse wr) {
                 logger.log(Level.FINE, "Message from findUserOrDie(): {0}", wr.getMessage());
             }
-
-            if (apiTokenUser == null) {
```

Comment:
Why is this check deleted?

Reply:
Because `apiTokenUser` will be `null` only if `apiTokenUser = findUserOrDie();` in try will throw exception. And handling of that is done in catch